pkgs/development/python-modules/furl/default.nix

```diff
@@ -26,7 +26,8 @@ buildPythonPackage rec {
   disabledTests = [
      # see https://github.com/gruns/furl/issues/121
     "join"
-    # The following tests are failing on some installations
+    # The following tests are failing due to AssertionErrors
+    # https://github.com/gruns/furl/issues/136
```

Comment:
This commit does not do, what it says it does.

Reply:
Actually, there is not need for disabling the tests. The failures were caused by https://bugs.python.org/issue42967 and in the issue tracker of `furl` was a patch attached.